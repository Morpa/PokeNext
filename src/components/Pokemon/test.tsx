import { renderWithTheme } from 'utils/tests/helpers'

import { Wrapper } from '.'

describe('<Pokemon />', () => {
  it('should render pokemon page correctly', () => {
    /* const { container } = renderWithTheme(
      <Wrapper>
        <Info>Pokemon</Info>
        <Back>Back</Back>
      </Wrapper>
    ) */
    /*  expect(container.firstChild).toHaveStyleRule('min-width', '35rem')

    expect(container.firstChild?.firstChild).toHaveStyleRule('color', '#FAFAFA')

    expect(container.firstChild).toMatchSnapshot() */
  })
})
