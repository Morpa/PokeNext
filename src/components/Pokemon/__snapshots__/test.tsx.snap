// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Pokemon /> should render pokemon page correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  min-width: 35rem;
  padding: 5rem;
}

.c1 {
  margin-bottom: 1rem;
  color: #FAFAFA;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c2 > a {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #FAFAFA;
}

@media (max-width:768px) {
  .c0 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

<div
  class="c0"
>
  <h3
    class="c1"
  >
    Pokemon
  </h3>
  <div
    class="c2"
  >
    Back
  </div>
</div>
`
